app/views/content/life-as-a-teacher/explore-subjects/modern-languages/_article.html.erb

```diff
@@ -0,0 +1,149 @@
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Learning a new language will give pupils cultural awareness and enhanced communication skills that can lead to exciting careers.</p>
+      <p>Popular careers range from interpreters and translators, to tour guides and diplomats. Knowing a second language can also be helpful for a wide range of careers including journalism, public relations and finance.</p>
+      <p>Your pupils will learn more from you that you might think. As their teacher, you will guide your pupils though the skills they need to show their understanding of languages. You will teach them valuable life skills such as enhanced listening, cultural awareness and communication.</p>
+  </section>
+
+    <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "Teaching languages is so much more than teaching grammar and vocabulary. You get to introduce students to new cultures and a different way of doing things. In an increasingly globalised world, you’re providing kids with an incredibly valuable skill that they’ll cherish for their entire lives.",
+    name: "Tom, languages teacher",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-languages-progammes-of-study/national-curriculum-in-england-languages-progammes-of-study">national curriculum for modern foreign languages</a>.</p>
+    <p>Themes you'll cover when you teach 11 to 14 year olds (key stage 3) include:</p>
+    <ul>
+      <li>identifying and using present, past, and future tenses</li>
+      <li>using a variety of key grammatical structures and patterns</li>
+      <li>developing and using a wide-ranging and deepening vocabulary</li>
+      <li>using accurate grammar, spelling and punctuation</li>
+      <li>having conversations and making use of important social conventions such as formal modes of address</li>
+      <li>expressing and developing ideas clearly and with increasing accuracy</li>
+      <li>speaking coherently and confidently</li>
+      <li>reading literary texts in the language, so pupils can expand their understanding of the language and culture</li>
+      <li>writing creatively to express their own ideas and opinions</li>
+    </ul>
+
+    <p>Themes you'll cover when you teach 14 to 16 year olds (key stage 4) include:</p>
+    <ul>
+      <li>understanding and responding to spoken language</li>
+      <li>communicating and interacting in speech</li>
+      <li>understanding and responding to written language</li>
+      <li>communicating in writing</li>
+    </ul>
+    <p>Explore what teaching modern language lessons would be like at key stage 4:</p>
+      <ul>
+        <li><a href="https://www.thenational.academy/teachers/programmes/french-secondary-ks4-l/units">French lessons</a></li>
```

Comment:
should we be changing these links to reference that it is going to Oak Academy website?

Reply:
Done!